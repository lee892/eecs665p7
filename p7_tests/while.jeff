fn : () int main {
    int j;
    int a;
    a = 5;
    j = 234 - 17 + a;
    while (j != 200 && a <= 5) {
        j = j - 1;
        if (j == 210) {
            a = 6;
        } else {
            console << "aaaaaah";
        }
        console << j;
        console << "\n";
    }
}