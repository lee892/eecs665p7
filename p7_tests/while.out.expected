aaaaaah221
aaaaaah220
aaaaaah219
aaaaaah218
aaaaaah217
aaaaaah216
aaaaaah215
aaaaaah214
aaaaaah213
aaaaaah212
aaaaaah211
210
